src/nile/cli.py

```diff
@@ -43,7 +43,7 @@ def network_option(f):
 def _validate_network(_ctx, _param, value):
     """Normalize network values."""
     # normalize goerli
-    if "goerli" in value or "testnet" in value:
+    if "goerli" == value or "testnet" in value:
         return "goerli"
     # normalize localhost
     if "localhost" in value or "127.0.0.1" in value:
```

Comment:
```suggestion
    if "127.0.0.1" == value:
```

I think accepting something like "localhost15", or "3127.0.0.12" as localhost is not the best approach, I would remove the "in" check.